index.js

```diff
@@ -28,10 +32,19 @@ export const startEvaluate = async ({
     cidsSeen.push(cid)
     if (cidsSeen.length > 1000) cidsSeen.shift()
 
+    if (!rounds.current) {
+      rounds.current = new RoundData(roundIndex)
+    } else if (rounds.current.index !== roundIndex) {
+      // This should never happen
+      throw new Error(
+        `Round index mismatch: ${rounds.current.index} !== ${roundIndex}`
+      )
```

Comment:
I've added round validation logic, but am not sure this is worth it. This is just one commit, so easy to revert. Wdyt @bajtos?

Reply:
I guess we should keep it if it's better for a measurement to be rejected than added to the wrong round.